src/reaper_osara.cpp

```diff
@@ -331,6 +331,31 @@ bool isItemSelected(MediaItem* item) {
 	return *(bool*)GetSetMediaItemInfo(item, "B_UISEL", NULL);
 }
 
+char* automationModeAsString(int mode) {
```

Comment:
Please make this `const char*`. We definitely don't want callers being able to modify these static strings. :)

Reply:
I don't know how I forgot that, sorry.